### PR TITLE
Websocket instead of client polling

### PR DESCRIPTION
**Pull request by capitanu**
In this PR the architecture of updating the webpage is modified: the client does not poll the server for updates anymore, instead the server sends the updates to the client when it has them, using a websocket.

### CODE REVIEW

File: make [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+
+# Install the server
+install() {
+    bin_path=$HOME/.local/bin/
+    mkdir -p $bin_path
+    cp -f build/mi $bin_path/mi; chmod +x $bin_path/mi
+    rm -rf $lib_path; cp -rf stdlib $lib_path
+}
```
Comment (aathn) [inline make]: This doesn't seem to belong here.

**Review by aathn — APPROVED** [review]
Looks good. I've tested it on Linux and it seems to work well.

File: make (hunk 1 above)
Comment (capitanu) [reply make]: Might have forgotten it there. Deleted and pushed again

**Comment by moaand** [comment]
I tested the code on Mac and it seems to work as before!

**Pull request closed without merging**